cclib/parser/molcasparser.py

```diff
@@ -45,8 +45,21 @@ def before_parsing(self):
         # Compile the dashes-and-or-spaces-only regex.
         self.re_dashes_and_spaces = re.compile('^[\s-]+$')
 
+    def _assing_gbasis_to_element(self, element, _array):
+        """Assign proper basis exponents and coefficients to element."""
+        mask = [element == possible_element
+                for possible_element in self.atomsymbols]
+        indices = [i for (i, x) in enumerate(mask) if x]
+        for index in indices:
+            self.gbasis[index] = _array
+
     def after_parsing(self):
-        pass
+        self.atomsymbols = [self.table.element[atomno]
+                                 for atomno in self.atomnos]
+        if hasattr(self, 'gbasis_array'):
```

Comment:
It is passing after this check, since the regression file don't have a `gbasis` block we don't need to parse for those anyways.

Reply:
In #632 I got rid of this, since it seems like an unnecessary complication. The regressions seem to run fine.